planner/core/logical_plan_test.go

```diff
@@ -1459,7 +1459,6 @@ func (s *testPlanSuite) TestNameResolver(c *C) {
 		{"delete a from (select * from t ) as a, t", "[planner:1288]The target table a of the DELETE is not updatable"},
 		{"delete b from (select * from t ) as a, t", "[planner:1109]Unknown table 'b' in MULTI DELETE"},
 		{"select '' as fakeCol from t group by values(fakeCol)", "[planner:1054]Unknown column '' in 'VALUES() function'"},
-		{"update t, (select * from t) as b set b.a = t.a", "[planner:1288]The target table b of the UPDATE is not updatable"},
```

Comment:
Why remove this test?

Reply:
this test here is not suitable, when build planner, we will `Preprocess` the AST to build some extract info such as `model.TableInfo` in `TableName`. This test `BuildLogicalPlan` directly, so that `model.TableInfo` in `TableName` is missing.